omisego-sdk/src/test/kotlin/co/omisego/omisego/OMGAPIClientTest.kt

```diff
@@ -157,7 +159,7 @@ class OMGAPIClientTest {
     fun `OMGAPIClient should be executed when API return success false correctly`() {
         expectedEx.expect(OMGAPIErrorException::class.java)
         val apiError = APIError(ErrorCode.CLIENT_INVALID_AUTH_SCHEME,
-                "The provided authentication scheme is not supported")
+            "The provided authentication scheme is not supported")
```

Comment:
Same here. Weirdly (intentionally?) Intellij/Android Studio doesn't allow the option to align multiline arguments like you can do in java, giving cases like this, where parameters are not aligned 😢

Reply:
Yeah it doesn't give me a warning, I need to keep it in mind 😢